### PR TITLE
Update readme & add wechat code

### PR DESCRIPTION
**Pull request by liuxiaocs7**
- fix email
- add weixin pic

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,5 +53,7 @@ The functions of this system include but are not limited to:
 
 ### Contact Us
 - [Github Issues](https://github.com/apache/incubator-hugegraph/issues): Feedback on usage issues and functional requirements (priority)
-- Feedback Email: [hugegraph@googlegroups.com](mailto:hugegraph@googlegroups.com)
-- WeChat public account: HugeGraph
+- Feedback Email: [dev@apache.hugegraph.org](mailto:dev@apache.hugegraph.org)
```
Comment (imbajin) [inline README.md:56]: wrong mail addr~  should。be hugegraph.apache.org？

File: README.md [hunk 2]
```diff
@@ -55,5 +53,7 @@ The functions of this system include but are not limited to:
...
+- WeChat public account: Apache HugeGraph, welcome to scan this QR code to follow us.
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <img src="./assets/images/weixin.jpg" width="180px">
```
Comment (imbajin) [inline README.md:59]: I will replace it with a better ui style

**Comment by zyxxoo** [comment]
image

File: README.md (hunk 1 above)
Comment (liuxiaocs7) [reply README.md]: fixed, thanks!

File: README.md (hunk 2 above)
Comment (liuxiaocs7) [reply README.md]: yeap, i am leaning towards style1 too!

**Comment by liuxiaocs7** [comment]
Hi, @imbajin and @zyxxoo, how do you think it is now, is there anything that needs to be improved?

image

**Review by imbajin — APPROVED** [review]
Looks well, could add/update it in all other repos later

**Comment by imbajin** [comment]
Also found outdated message in https://hugegraph.apache.org/docs/introduction/readme/ (EN & CN), could add a task in task lists for others to take

image

**Pull request merged**